students/Kristy_Martini/lesson03/list_lab.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+from slicing_lab import reverse
+
+if __name__ == "__main__":
+    # Series 1
+    # Create the initial list
+    fruits = ["Apples", "Pears", "Oranges", "Peaches"]   
+    # Ask the user for a new fruit and add it to the list
+    new_fruit = input("Please enter the name of a fruit: ")
+    fruits.append(new_fruit)
+    # Dispaly the list
+    print("List of fruits: ", fruits)
+    # Ask the user for a number
+    number_display = input("Please enter a number between 1 and 5 (inclusive): ")
+    # Display the numbered fruit in the list
+    print("Fruit at that value:", fruits[int(number_display)-1])
+    # Add a fruit to the beginning of the list using the '+' operator
+    fruits = ["Grapes"] + fruits
+    print("Newly added fruit:", fruits[0])
+    # Add another fruit using the insert() method, then display list
+    fruits.insert(0, "Bananas")
+    print("Updated list of fruit: ", fruits)
+    for item in fruits:
+        if item.startswith("P"):
```

Comment:
Recommend using upper() or lower() or something similar to catch case sensitive input.